README.md

```diff
@@ -82,7 +84,7 @@ For easier development and debugging purposes continue with the following steps:
 
 Now go and make a change to your plugin. It will be automatically applied to the demo project.
 
-NOTE: If you need to use a native library in your plugin or do some changes in Info.plist/AndroidManifest.xml, these cannot be applied to the demo project only by npm link. In such scenario, you need to use `tns plugin add ../src` from the `demo` so that the native libraries and changes in the above-mentioned files are applied in the demo. Then you can link again the code of your plugin in the demo by using `npm run plugin.link` from the `src`.
+**NOTE**: Any changes that you need to make in a native library used in your plugin or in any other files inside `src/platforms` directory such as Info.plist or AndroidManifest.xml can't be directly reflected in the demo app. You need to use `npm run demo.reset` and run the application again.
```

Comment:
As far as I know, even if the aar inside src/platforms/android is rebuilt, the manifest that is already in demo/platforms/android is not updated. This is why we need to clear demo/platforms so that on the next build the correct manifest is taken.